sdk/keyvault/azure-security-keyvault-keys/test/key_client_test_live.cpp

```diff
@@ -121,3 +122,36 @@ TEST_F(KeyVaultClientTest, CreateKeyWithTags)
     };
   }
 }
+
+// Test Key Delete.
+// The test works for either soft-delete or not, but for non soft-delete, the LRO is completed as
+// soon as the Operation returns.
+TEST_F(KeyVaultClientTest, DeleteKey)
```

Comment:
Add tests for Deleting an invalid key, deleting a key that was already deleted, add tests for Poll in addition to poll until done.

Reply:
done